packages/generator/src/templateHelpers.ts

```diff
@@ -96,11 +96,11 @@ export function extractTypeFromPayload(payload: model.domain.Payload): string {
   if ((payload.schema as model.domain.UnionShape).anyOf !== undefined) {
     const union: string[] = [];
     (payload.schema as model.domain.UnionShape).anyOf.forEach(element => {
-      union.push(element.name.value() + "T");
+      union.push(`types.${element.name.value()}`);
```

Comment:
So an important part is that we move this logic outside the templateHelpers. Both solutions (adding a T or making a 'types' namespace) are specific to commerce-sdk. These functions are moving to raml-toolkit and shouldn't know about either of those things. We to move this logic to the templates where these are being called.